pipeline/trusted-resources/pkg/trustedtask/trustedtask_test.go

```diff
@@ -0,0 +1,164 @@
+package trustedtask
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	faketekton "github.com/tektoncd/pipeline/pkg/client/clientset/versioned/fake"
+	"github.com/tektoncd/pipeline/test/diff"
+	"go.uber.org/zap/zaptest"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	fakek8s "k8s.io/client-go/kubernetes/fake"
+	"knative.dev/pkg/logging"
+)
+
+func init() {
+	os.Setenv("SYSTEM_NAMESPACE", nameSpace)
+	os.Setenv("WEBHOOK_SERVICEACCOUNT_NAME", serviceAccount)
+}
+
+func Test_verifyTask(t *testing.T) {
+	ctx := logging.WithLogger(context.Background(), zaptest.NewLogger(t).Sugar())
+	k8sClient := fakek8s.NewSimpleClientset(sa)
+	tektonClient := faketekton.NewSimpleClientset()
+	signer, secretpath, _ := getSignerFromFile(t, ctx, k8sClient)
```

Comment:
It looks like we're ignoring the error everytime. Looking at the impl it looks like we never return an error, so can we removing this from the return vals?

Reply:
good catch! think perhaps I can add the err on the return "load", so populating the errors if any out.

Reply:
considering this helper function is under a test file. the error could be thrown in the func, so no need to pass to the return vals. 

agree with the original feedback. updated to remove the error in a follow-up PR
https://github.com/tektoncd/experimental/pull/888